files/import_eoni_from_s3.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+set -x
+
+
+LATEST_FILE=$(aws s3 ls s3://eoni-data.wheredoivote.co.uk/production_data/ | sort | tail -n1 | rev | cut -d' ' -f1 | rev)
+SRCDIR='/tmp/eoni_production_data'
+mkdir -p $SRCDIR
+
+aws s3 sync s3://eoni-data.wheredoivote.co.uk/production_data/ $SRCDIR --exclude '*' --include "*${LATEST_FILE}"
```

Comment:
As we know the file name, I think this can be `aws s3 cp s3://eoni-data.wheredoivote.co.uk/production_data/${LATEST_FILE} $SRCDIR` ?

Reply:
Yep - I made it a sync when I didn't know updates would be nightly - i.e. to save re-downloading.